NEWS.md

```diff
@@ -2,12 +2,46 @@
 
 ## New Features
 
-
 ## Updates of Existing Functions
 
+- For the `set_values_to` argument expressions are accepted now. This affects
```

Comment:
Should we define what we consider an expression to be?

Reply:
I am not sure how we could define it in a concise and clear way (if you have a good definition, let me know). I have added an example. Does this help?